code/modules/antagonists/cult/cult.dm

```diff
@@ -158,11 +155,8 @@
 	current.clear_alert("bloodsense")
 	if(ishuman(current))
 		var/mob/living/carbon/human/H = current
-		H.eye_color = initial(H.eye_color)
-		H.dna.update_ui_block(DNA_EYE_COLOR_BLOCK)
+		SEND_SIGNAL(H, COMSIG_CULT_DECONVERT)
```

Comment:
maybe this can just be an 'update_level' signal instead of only having 'raise_level'